src/commands/BugCommand.ts

```diff
@@ -30,39 +40,26 @@ export default class BugCommand extends PrefixCommand {
 			embed = await mention.getEmbed();
 		} catch ( err ) {
 			try {
-				await message.channel.send( err );
+				await interaction.reply( { content: err, ephemeral: true } );
```

Comment:
The `err` variable we get here usually does not result in an error message that can just be sent as a reply. This results in the broad error message "An error occurred while running this command" always if JIRA returns an error. E.g. private bug report, too large ticket number, deleted bug report, etc.
This might be out of scope for this PR since error handling with MojiraBot is a mess everywhere at the moment. Might be good for a follow-up PR though.